source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -33,6 +33,8 @@ namespace
     // Lists of known metadata which are handled explicitly by import/export.
     static const RtTokenSet nodedefMetadata     = { RtToken("name"), RtToken("type"), RtToken("node") };
     static const RtTokenSet attrMetadata        = { RtToken("name"), RtToken("type"), RtToken("value"), RtToken("nodename"), RtToken("output") };
+    static const RtTokenSet inputMetadata       = { RtToken("name"), RtToken("type"), RtToken("value"), RtToken("nodename"), RtToken("output"),
```

Comment:
Would kind of be nice if there was an opt in version of this instead of having to opt out of everything, but this seems to fix the issue.

Reply:
Yes. Agreed.